canonical/snapcraft.io
# Show a no store page is user isn't in store

## Done
Show a no stores page to a user if they are not in a store or it does not exist

## How to QA
- Go to https://snapcraft-io-4129.demos.haus/admin/madeupstore/snaps
- Check that you see a "Store not found" page
- Go to https://snapcraft-io-4129.demos.haus/admin/madeupstore/members
- Check that you see a "Store not found" page
- Go to https://snapcraft-io-4129.demos.haus/admin/madeupstore/settings
- Check that you see a "Store not found" page

## Issue / Card
Fixes #4112
